Dockerfile

```diff
@@ -32,14 +32,16 @@
 # /////////////////////////////////////////////////////////////////////////////
 # Stage 1 - Acquire the CARMA source as well as any extra packages
 # /////////////////////////////////////////////////////////////////////////////
-
-FROM usdotfhwastoldev/autoware.ai:develop AS base-image
+ARG DOCKER_ORG="usdotfhwastoldev"
+ARG DOCKER_TAG="develop"
+FROM ${DOCKER_ORG}/autoware.ai:${DOCKER_TAG} as base-image
+ARG GIT_BRANCH="develop" 
```

Comment:
Now I'm less sure about my comment regarding ENV on the other PR, if this works I'm not clear on why the other one seems to return empty string when eval'ing ${GIT_BRANCH} in checkout.bash. Hmm.